File: Practice 7/Krymov/Hse/Hse.DbGenerator/Models/Olympic.cs
using System;
using System.Collections.Generic;

namespace Hse.DbGenerator.Models
{
    public class Olympic
    {
        public const int OlympicIdMaxLength = 7;
        public string OlympicId { get; set; }
        
        public const int CountryIdMaxLength = 3;
        public string CountryId { get; set; }

        public const int CityMaxLength = 50;
        public string City { get; set; }
        
        public int Year { get; set; }
        
        public DateTime StartDate { get; set; }
        
        public DateTime EndDate { get; set; }

        public Country Country { get; set; }
        
        public ICollection<Event> Events { get; set; }
    }
}

File: Practice 7/Kalmykov/db-filler/src/main/kotlin/com/example/databasefiller/models/Olympic.kt
package com.example.databasefiller.models

import com.example.databasefiller.models.country.Country
import java.sql.Date

const val MIN_YEAR = 1980
const val MAX_YEAR = 2010

data class Olympic(
    val olympicId: String,
    val country: Country,
    val city: String,
    val year: Int,
    val startDate: Date,
    val endDate: Date
)
